File: exercpythonbrasil/1.exerc_sequencial/exerc13.py
"""
Tendo como dado de entrada a altura (h) de uma pessoa, construa um algoritmo
que calcule seu peso ideal, utilizando as seguintes fórmulas:
A. Para homens: (72.7*h) - 58
B. Para mulheres: (62.1*h) - 44.7
"""
altura = float(input("Informe a altura: "))
while True:
    sexo = input("H - Homem ou M - Mulher: ")
    if sexo == 'h' or sexo == 'H' or sexo == 'm' or sexo == 'M':
        break
    else:
        print('Sexo não reconhecido. Informe novamente.')
if sexo == 'h' or sexo == 'H':
    print((72.7*altura)-58)
else:
    print((62.1*altura)-44.7)

File: exercicios_C/1.exercSequenciais/exerc13.c
#include <stdio.h>
#include <string.h>

void main(){
    char gen = "";
    float altura = 0;
    do{
        fflush(stdin);
        printf("informe o genero: ");
        gen = getchar();
    }while(toupper(gen) != 'H' && toupper(gen) != 'M');
    printf("informe a altura: ");
    scanf("%f", &altura);
    if (toupper(gen) == 'H'){
        printf("peso ideal de homem: %0.2f KG.", (float)(72 * altura) - 58);
    }else
    printf("peso ideal de mulher: %0.2f KG.", (float)(62.1 * altura) - 44.7);
}
